src/LuaPrinter.ts

```diff
@@ -351,18 +376,24 @@ export class LuaPrinter {
         return this.createSourceNode(statement, [this.indent("::"), statement.name, "::\n"]);
     }
 
-    private printReturnStatement(statement: tstl.ReturnStatement): SourceNode {
+    private printReturnStatement(statement: tstl.ReturnStatement, inline?: boolean): SourceNode {
```

Comment:
What about this inline argument, would it be possible to remove this from the interface?

Reply:
Yeah - I'll refactor some stuff around to make that work